src/ledger.ts

```diff
@@ -76,6 +78,34 @@ export class LedgerCanister {
     );
   };
 
+  public createSubaccount = async ({ name }: { name: string }) => {
+    const service = Actor.createActor<NNSDappService>(certifiedIdlFactory, {
+      agent: this.agent,
+      canisterId: this.canisterId,
+    });
+
+    const counter = 0;
+    let response = await service.create_sub_account(name);
+    while (response.AccountNotFound === null && counter < 2) {
```

Comment:
- do we really still need that weird `while`? according comment it used too but do we still need it?

- what's `2`? constant + comment

Reply:
More than the while, I needed to call `add_account`. But I don't understand why. I added the while because I learned that it fixed the issue. Do you know who could help us answer this question?